input/fsh/DEF_CodeSystems.fsh

```diff
@@ -1,87 +1,23 @@
-Description: "Unit to characterize the Energy spectrum used in Radiotherapy.
-For electrons, the maximum energy is given in MeV.
-For photons, the maximum acceleration voltage is given in MV or kV, although those are not units of energy."
-* ^copyright = "This value set includes content from The Unified Code for Units of Measure (UCUM), also known as the UCUM Specification which is copyright ©1999-2021, Regenstrief Institute, Inc. All rights reserved."
-//* ^copyright = "This value set includes content from The Unified Code for Units of Measure (UCUM), also known as the \“UCUM Specification,\” which is copyright ©1999-2021, Regenstrief Institute, Inc. All rights reserved."
-* ^status = #draft
-* ^experimental = false
-* UCUM#MeV "megaelectronvolt"
-* UCUM#MV "megavolt"
-* UCUM#kV "kilovolt"
-
-CodeSystem: ReasonForRevisionOrAdaptationCS
-Id: codex-radiotherapy-reason-for-revision-or-adaptation-cs
-Title: "Reason for Revision or Adaptation Code System"
-Description: "The reason a planned or prescribed radiotherapy treatment was revised, superceded, or adapted."
-* ^status = #draft
-* ^caseSensitive = true
-* ^experimental = false
-* #treatmentintentchange	"Changes to Treatment Intent (finding)" 	"new concept under 373856008 |Reason for change in radiotherapy course (finding)"
-* #targetchange  "Changes to Target Anatomy (finding)" "new concept under 373856008 |Reason for change in radiotherapy course (finding)."
-* #organatriskchange  "Changes to Organ at Risk Anatomy (finding)" "new concept under 373856008 |Reason for change in radiotherapy course (finding)."
-
-CodeSystem: ApplicatorTypesCS
-Id: codex-radiotherapy-applicator-types-cs
-Title: "Brachytherapy Applicator Types Code System"
-Description: "Brachytherapy Applicator Types  -- need to request codes"
-* ^status = #draft
-* ^caseSensitive = true
-* ^experimental = false
-* #ring	"Ring, device (physical object)" 	"new concept under 228766009 |Form of brachytherapy source (physical object)"
-* #tandem "Tandem, device (physical object)" "(small metal tube)  new code child of 228775006 |Tubes source (physical object)."
-* #ovoid 	"Ovoid, device (physical object)" 	"(round hollow metal holders that are placed adjacent to cervix) new concept under 228766009 |Form of brachytherapy source (physical object)|"
-* #cylinder  "Vaginal Cylinder, device (physical object)" "A vaginal cylinder is made of plastic and looks like a large tampon with a hollow center. It is placed into the vagina and may be kept in place with gauze, balloons, or a special undergarment.  )  define a new concept under 228766009 |Form of brachytherapy source (physical object)| ."
+//* Requested 10/2022
+* #USCRS-34602  "Radiotherapy Treatment Plan (therapy/regime)" "Radiotherapy Treatment Plan (therapy/regime)"
+//* These haven't been requested
+* #USCRS-34603	"Changes to Treatment Intent (finding)" "new concept under 373856008 |Reason for change in radiotherapy course (finding)" //#treatmentintentchange
```

Comment:
See latest code display strings in Sharon's mail from the workgroup call: "Change in treatment intent", "Change in target", "Change in organ at risk". We should already use lower case as usual in SNOMED CT.

Reply:
done